### PR TITLE
apply dampingFactor to dollyOut/dollyIn in OrbitControls.js

### PR DESCRIPTION
**Pull request by binary-husky**
Related issue: In orbit control, the dolly in / dolly out was not controlled by damping

**Description**
apply dampingFactor to dollyOut/dollyIn,
making zoom in/ zoom out looks more smooth.

### CODE REVIEW

**Comment by mrdoob** [comment]
Renamed some variables and cleaned up the code.

Should be it be `dollyFrom` and `dollyTo` instead?

**Comment by WestLangley** [comment]
mac already has built-in damping. This PR changes the feel of that. It seems less-responsive.

File: examples/jsm/controls/OrbitControls.js [hunk 1]
```diff
@@ -224,8 +224,28 @@ class OrbitControls extends EventDispatcher {
 
 				spherical.makeSafe();
 
+				if ( scope.enableDamping === true ) {
+
+					if ( scale !== 1 ) {
+
+						zoomFrom = spherical.radius;
+						zoomTo = spherical.radius * scale;
+						zoomDelta = ( zoomTo - zoomFrom ) * 3; // x3 because slow
```
Comment (WestLangley) [inline examples/jsm/controls/OrbitControls.js:233]: Is this magic number hardware-specific?

Comment (binary-husky) [reply examples/jsm/controls/OrbitControls.js:233]: Tested on windows, "x3" is chosen to make the dollying more comfortable ~

Comment (binary-husky) [reply examples/jsm/controls/OrbitControls.js:233]: I'm not sure about the mechanism of macOS's built-in damping, does it means that 'handleMouseMoveDolly' is called multiple times during a single mouse wheel movement?

Comment (WestLangley) [reply examples/jsm/controls/OrbitControls.js]: No. In MacOS, scrolling inertia is built into the OS. This PR is messing that up.

**Comment by Mugen87** [comment]
Closing due to https://github.com/mrdoob/three.js/pull/23628#discussion_r820104182.

**Pull request closed without merging**